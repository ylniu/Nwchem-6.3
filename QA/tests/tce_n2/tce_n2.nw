start tce_n2
 
echo
 
geometry units bohr
  symmetry d2h
  n 0 0   -1.034
  n 0 0    1.034
end
 
basis
 N   S
   9046.0000000      0.70000000000E-03 -0.15300000000E-03
   1357.0000000      0.53890000000E-02 -0.12080000000E-02
   309.30000000      0.27406000000E-01 -0.59920000000E-02
   87.730000000      0.10320700000     -0.24544000000E-01
   28.560000000      0.27872300000     -0.67459000000E-01
   10.210000000      0.44854000000     -0.15807800000
   3.8380000000      0.27823800000     -0.12183100000
  0.74660000000      0.15440000000E-01  0.54900300000
 N   S
  0.22480000000       1.0000000000
 N   P
   13.550000000      0.39919000000E-01
   2.9170000000      0.21716900000
  0.79730000000      0.51031900000
 N   P
  0.21850000000       1.0000000000
 N   D
  0.81700000000       1.0000000000
end
 
scf
thresh 1.0e-10
tol2e 1.0e-10
singlet
rhf
end
 
tce
freeze atomic
ccsd
end
 
task tce energy
 
