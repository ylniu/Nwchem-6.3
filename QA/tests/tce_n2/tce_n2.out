 
Processor list
 
cu05n[83-84]
 
ARMCI configured for 2 cluster nodes. Network protocol is 'OpenIB Verbs API'.
All connections between all procs tested: SUCCESS
 argument  1 = /mscf/home/d3p852/nwchem-5.1.1/QA/tests/tce_n2/tce_n2.nw



============================== echo of input deck ==============================
start tce_n2

echo

geometry units bohr
  symmetry d2h
  n 0 0   -1.034
  n 0 0    1.034
end

basis
 N   S
   9046.0000000      0.70000000000E-03 -0.15300000000E-03
   1357.0000000      0.53890000000E-02 -0.12080000000E-02
   309.30000000      0.27406000000E-01 -0.59920000000E-02
   87.730000000      0.10320700000     -0.24544000000E-01
   28.560000000      0.27872300000     -0.67459000000E-01
   10.210000000      0.44854000000     -0.15807800000
   3.8380000000      0.27823800000     -0.12183100000
  0.74660000000      0.15440000000E-01  0.54900300000
 N   S
  0.22480000000       1.0000000000
 N   P
   13.550000000      0.39919000000E-01
   2.9170000000      0.21716900000
  0.79730000000      0.51031900000
 N   P
  0.21850000000       1.0000000000
 N   D
  0.81700000000       1.0000000000
end

scf
thresh 1.0e-10
tol2e 1.0e-10
singlet
rhf
end

tce
freeze atomic
ccsd
end

task tce energy

================================================================================


                                         
                                         


              Northwest Computational Chemistry Package (NWChem) 6.0
              ------------------------------------------------------


                    Environmental Molecular Sciences Laboratory
                       Pacific Northwest National Laboratory
                                Richland, WA 99352

                              Copyright (c) 1994-2010
                       Pacific Northwest National Laboratory
                            Battelle Memorial Institute

             NWChem is an open-source computational chemistry package
                        distributed under the terms of the
                      Educational Community License (ECL) 2.0
             A copy of the license is included with this distribution
                              in the LICENSE.TXT file

                                  ACKNOWLEDGMENT
                                  --------------

            This software and its documentation were developed at the
            EMSL at Pacific Northwest National Laboratory, a multiprogram
            national laboratory, operated for the U.S. Department of Energy
            by Battelle under Contract Number DE-AC05-76RL01830. Support
            for this work was provided by the Department of Energy Office
            of Biological and Environmental Research, Office of Basic
            Energy Sciences, and the Office of Advanced Scientific Computing.


           Job information
           ---------------

    hostname      = cu5n83
    program       = /scratch/nwchem
    date          = Fri Oct 29 11:41:34 2010

    compiled      = Thu_Oct_28_07:10:53_2010
    source        = /home/scicons/user/kurt/nwchem-6.0-release-pgf90-final/
    nwchem branch = 6.0
    input         = /mscf/home/d3p852/nwchem-5.1.1/QA/tests/tce_n2/tce_n2.nw
    prefix        = tce_n2.
    data base     = ./tce_n2.db
    status        = startup
    nproc         =        8
    time left     =   1764s



           Memory information
           ------------------

    heap     =  104857601 doubles =    800.0 Mbytes
    stack    =  104857601 doubles =    800.0 Mbytes
    global   =  209715200 doubles =   1600.0 Mbytes (distinct from heap & stack)
    total    =  419430402 doubles =   3200.0 Mbytes
    verify   = yes
    hardfail = no 


           Directory information
           ---------------------

  0 permanent = .
  0 scratch   = .




                                NWChem Input Module
                                -------------------


 Turning off AUTOSYM since
 SYMMETRY directive was detected!


          ------
          auto-z
          ------


                             Geometry "geometry" -> ""
                             -------------------------

 Output coordinates in a.u. (scale by  1.000000000 to convert to a.u.)

  No.       Tag          Charge          X              Y              Z
 ---- ---------------- ---------- -------------- -------------- --------------
    1 n                    7.0000     0.00000000     0.00000000    -1.03400000
    2 n                    7.0000     0.00000000     0.00000000     1.03400000

      Atomic Mass 
      ----------- 

      n                 14.003070


 Effective nuclear repulsion energy (a.u.)      23.6943907157

            Nuclear Dipole moment (a.u.) 
            ----------------------------
        X                 Y               Z
 ---------------- ---------------- ----------------
     0.0000000000     0.0000000000     0.0000000000

      Symmetry information
      --------------------

 Group name             D2h       
 Group number             26
 Group order               8
 No. of unique centers     1

      Symmetry unique atoms

     1



                                Z-matrix (autoz)
                                -------- 

 Units are Angstrom for bonds and degrees for angles

      Type          Name      I     J     K     L     M      Value
      ----------- --------  ----- ----- ----- ----- ----- ----------
    1 Stretch                  1     2                       1.09434


            XYZ format geometry
            -------------------
     2
 geometry
 n                     0.00000000     0.00000000    -0.54716928
 n                     0.00000000     0.00000000     0.54716928

 ==============================================================================
                                internuclear distances
 ------------------------------------------------------------------------------
       center one      |      center two      | atomic units |       a.u.
 ------------------------------------------------------------------------------
    2 n                |   1 n                |     2.06800  |     2.06800
 ------------------------------------------------------------------------------
                         number of included internuclear distances:          1
 ==============================================================================



  library name resolved from: environment
  library file name is: <
 /mscf/scicons/apps/nwchem-6.0.oct19//src/basis/libraries/>
  
                      Basis "ao basis" -> "" (cartesian)
                      -----
  N (Nitrogen)
  ------------
            Exponent  Coefficients 
       -------------- ---------------------------------------------------------
  1 S  9.04600000E+03  0.000700
  1 S  1.35700000E+03  0.005389
  1 S  3.09300000E+02  0.027406
  1 S  8.77300000E+01  0.103207
  1 S  2.85600000E+01  0.278723
  1 S  1.02100000E+01  0.448540
  1 S  3.83800000E+00  0.278238
  1 S  7.46600000E-01  0.015440

  2 S  9.04600000E+03 -0.000153
  2 S  1.35700000E+03 -0.001208
  2 S  3.09300000E+02 -0.005992
  2 S  8.77300000E+01 -0.024544
  2 S  2.85600000E+01 -0.067459
  2 S  1.02100000E+01 -0.158078
  2 S  3.83800000E+00 -0.121831
  2 S  7.46600000E-01  0.549003

  3 S  2.24800000E-01  1.000000

  4 P  1.35500000E+01  0.039919
  4 P  2.91700000E+00  0.217169
  4 P  7.97300000E-01  0.510319

  5 P  2.18500000E-01  1.000000

  6 D  8.17000000E-01  1.000000



 Summary of "ao basis" -> "" (cartesian)
 ------------------------------------------------------------------------------
       Tag                 Description            Shells   Functions and Types
 ---------------- ------------------------------  ------  ---------------------
 N                       user specified              6       15   3s2p1d


                                 NWChem SCF Module
                                 -----------------



  ao basis        = "ao basis"
  functions       =    30
  atoms           =     2
  closed shells   =     7
  open shells     =     0
  charge          =   0.00
  wavefunction    = RHF 
  input vectors   = atomic
  output vectors  = ./tce_n2.movecs
  use symmetry    = T
  symmetry adapt  = T


 Summary of "ao basis" -> "ao basis" (cartesian)
 ------------------------------------------------------------------------------
       Tag                 Description            Shells   Functions and Types
 ---------------- ------------------------------  ------  ---------------------
 N                       user specified              6       15   3s2p1d


      Symmetry analysis of basis
      --------------------------

        ag          8
        au          1
        b1g         1
        b1u         8
        b2g         3
        b2u         3
        b3g         3
        b3u         3


 Forming initial guess at       1.8s


      Superposition of Atomic Density Guess
      -------------------------------------

 Sum of atomic energies:        -108.60004629

      Non-variational initial energy
      ------------------------------

 Total energy =    -109.123177
 1-e energy   =    -194.140722
 2-e energy   =      61.323154
 HOMO         =      -0.418440
 LUMO         =       0.037253


      Symmetry analysis of molecular orbitals - initial
      -------------------------------------------------


 !! scf_movecs_sym_adapt:   16 vectors were symmetry contaminated

  Symmetry fudging

 !! scf_movecs_sym_adapt:   14 vectors were symmetry contaminated

  Numbering of irreducible representations: 

     1 ag          2 au          3 b1g         4 b1u         5 b2g     
     6 b2u         7 b3g         8 b3u     

  Orbital symmetries:

     1 ag          2 b1u         3 ag          4 b1u         5 b3u     
     6 b2u         7 ag          8 b2g         9 b3g        10 b1u     
    11 ag         12 b3u        13 b2u        14 ag         15 b3g     
    16 b2g        17 b1u     


 Starting SCF solution at       1.9s



 ----------------------------------------------
         Quadratically convergent ROHF

 Convergence threshold     :          1.000E-10
 Maximum no. of iterations :           30
 Final Fock-matrix accuracy:          1.000E-10
 ----------------------------------------------


 #quartets = 2.094D+03 #integrals = 2.325D+04 #direct =  0.0% #cached =100.0%


 Integral file          = ./tce_n2.aoints.0
 Record size in doubles =  65536        No. of integs per rec  =  43688
 Max. records in memory =      2        Max. records in file   = 169376
 No. of bits per label  =      8        No. of bits per value  =     64


File balance: exchanges=     0  moved=     0  time=   0.0


              iter       energy          gnorm     gmax       time
             ----- ------------------- --------- --------- --------
                 1     -108.9433050655  4.28D-01  1.87D-01      0.6
                 2     -108.9544198129  1.04D-01  4.84D-02      0.6
                 3     -108.9551520698  1.60D-03  9.60D-04      0.7
                 4     -108.9551522081  5.12D-07  2.52D-07      0.7
                 5     -108.9551522081  1.94D-12  1.14D-12      0.8


       Final RHF  results 
       ------------------ 

         Total SCF energy =   -108.955152208117
      One-electron energy =   -194.370539428696
      Two-electron energy =     61.720996504912
 Nuclear repulsion energy =     23.694390715667

        Time for solution =      0.2s



       Symmetry analysis of molecular orbitals - final
       -----------------------------------------------

  Numbering of irreducible representations: 

     1 ag          2 au          3 b1g         4 b1u         5 b2g     
     6 b2u         7 b3g         8 b3u     

  Orbital symmetries:

     1 ag          2 b1u         3 ag          4 b1u         5 ag      
     6 b3u         7 b2u         8 b3g         9 b2g        10 b1u     
    11 ag         12 b3u        13 b2u        14 ag         15 b2g     
    16 b3g        17 b1u     

             Final eigenvalues
             -----------------

              1      
    1  -15.6866
    2  -15.6831
    3   -1.4740
    4   -0.7735
    5   -0.6271
    6   -0.6099
    7   -0.6099
    8    0.1769
    9    0.1769
   10    0.5941
   11    0.8030
   12    0.8709
   13    0.8709
   14    0.8816
   15    1.0514
   16    1.0514
   17    1.1108

                       ROHF Final Molecular Orbital Analysis
                       -------------------------------------

 Vector    2  Occ=2.000000D+00  E=-1.568313D+01  Symmetry=b1u
              MO Center= -3.1D-22,  1.4D-20, -5.7D-13, r^2= 3.2D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     1      0.708251  1 N  s                 16     -0.708251  2 N  s          

 Vector    3  Occ=2.000000D+00  E=-1.474012D+00  Symmetry=ag
              MO Center= -2.8D-34, -6.9D-34, -7.4D-17, r^2= 4.4D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    17      0.331729  2 N  s                  2      0.331729  1 N  s          
     6      0.218318  1 N  pz                21     -0.218318  2 N  pz         
    18      0.187745  2 N  s                  3      0.187745  1 N  s          

 Vector    4  Occ=2.000000D+00  E=-7.735351D-01  Symmetry=b1u
              MO Center= -4.6D-38,  6.2D-37, -8.2D-17, r^2= 1.3D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     3      0.435587  1 N  s                 18     -0.435587  2 N  s          
     2      0.326753  1 N  s                 17     -0.326753  2 N  s          
     6     -0.217650  1 N  pz                21     -0.217650  2 N  pz         

 Vector    5  Occ=2.000000D+00  E=-6.271188D-01  Symmetry=ag
              MO Center=  3.9D-21,  7.5D-21,  2.0D-16, r^2= 1.3D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    21      0.453506  2 N  pz                 6     -0.453506  1 N  pz         
     3      0.342322  1 N  s                 18      0.342322  2 N  s          
    24      0.211546  2 N  pz                 9     -0.211546  1 N  pz         

 Vector    6  Occ=2.000000D+00  E=-6.099201D-01  Symmetry=b3u
              MO Center=  6.2D-19,  1.3D-33, -3.5D-18, r^2= 9.0D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     4      0.426825  1 N  px                19      0.426825  2 N  px         
     7      0.256071  1 N  px                22      0.256071  2 N  px         

 Vector    7  Occ=2.000000D+00  E=-6.099201D-01  Symmetry=b2u
              MO Center=  2.8D-33,  8.3D-21,  1.1D-36, r^2= 9.0D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    20      0.426825  2 N  py                 5      0.426825  1 N  py         
     8      0.256071  1 N  py                23      0.256071  2 N  py         

 Vector    8  Occ=0.000000D+00  E= 1.769483D-01  Symmetry=b3g
              MO Center=  2.7D-33,  5.0D-18,  3.7D-17, r^2= 1.4D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     8      0.680198  1 N  py                23     -0.680198  2 N  py         
    20     -0.427390  2 N  py                 5      0.427390  1 N  py         

 Vector    9  Occ=0.000000D+00  E= 1.769483D-01  Symmetry=b2g
              MO Center= -6.3D-19, -2.8D-34,  2.3D-17, r^2= 1.4D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     7      0.680198  1 N  px                22     -0.680198  2 N  px         
    19     -0.427390  2 N  px                 4      0.427390  1 N  px         

 Vector   10  Occ=0.000000D+00  E= 5.940605D-01  Symmetry=b1u
              MO Center=  3.7D-19,  1.8D-19,  4.5D-17, r^2= 3.2D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    18      3.842185  2 N  s                  3     -3.842185  1 N  s          
     9     -2.603045  1 N  pz                24     -2.603045  2 N  pz         
     2     -0.177647  1 N  s                 17      0.177647  2 N  s          

 Vector   11  Occ=0.000000D+00  E= 8.030294D-01  Symmetry=ag
              MO Center=  4.6D-21,  1.4D-20,  6.9D-16, r^2= 1.8D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    24      0.844780  2 N  pz                 9     -0.844780  1 N  pz         
    17      0.542400  2 N  s                  2      0.542400  1 N  s          
    18     -0.456897  2 N  s                  3     -0.456897  1 N  s          
    21     -0.332542  2 N  pz                 6      0.332542  1 N  pz         
     1      0.282062  1 N  s                 16      0.282062  2 N  s          

 Vector   12  Occ=0.000000D+00  E= 8.708526D-01  Symmetry=b3u
              MO Center=  2.6D-19, -1.1D-33, -3.7D-17, r^2= 1.7D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     4      0.641116  1 N  px                19      0.641116  2 N  px         
     7     -0.619820  1 N  px                22     -0.619820  2 N  px         
    12      0.179418  1 N  dxz               27     -0.179418  2 N  dxz        

 Vector   13  Occ=0.000000D+00  E= 8.708526D-01  Symmetry=b2u
              MO Center=  5.7D-35,  1.1D-18,  5.2D-18, r^2= 1.7D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     5      0.641116  1 N  py                20      0.641116  2 N  py         
     8     -0.619820  1 N  py                23     -0.619820  2 N  py         
    14      0.179418  1 N  dyz               29     -0.179418  2 N  dyz        

 Vector   14  Occ=0.000000D+00  E= 8.815656D-01  Symmetry=ag
              MO Center= -4.5D-19, -1.1D-18, -1.5D-15, r^2= 2.4D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    18      1.067905  2 N  s                  3      1.067905  1 N  s          
     2     -0.697744  1 N  s                 17     -0.697744  2 N  s          
     9     -0.470610  1 N  pz                24      0.470610  2 N  pz         
    21     -0.416065  2 N  pz                 6      0.416065  1 N  pz         
    16     -0.317978  2 N  s                  1     -0.317978  1 N  s          

 Vector   15  Occ=0.000000D+00  E= 1.051362D+00  Symmetry=b2g
              MO Center=  3.0D-18, -1.0D-34, -1.5D-17, r^2= 2.0D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     7      1.159530  1 N  px                22     -1.159530  2 N  px         
     4     -0.732890  1 N  px                19      0.732890  2 N  px         

 Vector   16  Occ=0.000000D+00  E= 1.051362D+00  Symmetry=b3g
              MO Center=  4.0D-36, -2.3D-19, -3.9D-17, r^2= 2.0D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     8      1.159530  1 N  py                23     -1.159530  2 N  py         
    20      0.732890  2 N  py                 5     -0.732890  1 N  py         

 Vector   17  Occ=0.000000D+00  E= 1.110833D+00  Symmetry=b1u
              MO Center= -6.4D-18, -6.1D-18,  3.1D-16, r^2= 1.3D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     3     -1.860459  1 N  s                 18      1.860459  2 N  s          
     6     -0.672798  1 N  pz                21     -0.672798  2 N  pz         
    15      0.375322  1 N  dzz               30     -0.375322  2 N  dzz        
    17     -0.335897  2 N  s                  2      0.335897  1 N  s          
    24     -0.299094  2 N  pz                 9     -0.299094  1 N  pz         


 center of mass
 --------------
 x =   0.00000000 y =   0.00000000 z =   0.00000000

 moments of inertia (a.u.)
 ------------------
          29.942932617840           0.000000000000           0.000000000000
           0.000000000000          29.942932617840           0.000000000000
           0.000000000000           0.000000000000           0.000000000000

  Mulliken analysis of the total density
  --------------------------------------

    Atom       Charge   Shell Charges
 -----------   ------   -------------------------------------------------------
    1 N    7     7.00   2.00  0.85  0.90  2.13  1.05  0.06
    2 N    7     7.00   2.00  0.85  0.90  2.13  1.05  0.06

       Multipole analysis of the density wrt the origin
       ------------------------------------------------

     L   x y z        total         open         nuclear
     -   - - -        -----         ----         -------
     0   0 0 0      0.000000      0.000000     14.000000

     1   1 0 0      0.000000      0.000000      0.000000
     1   0 1 0      0.000000      0.000000      0.000000
     1   0 0 1      0.000000      0.000000      0.000000

     2   2 0 0     -7.541334      0.000000      0.000000
     2   1 1 0      0.000000      0.000000      0.000000
     2   1 0 1      0.000000      0.000000      0.000000
     2   0 2 0     -7.541334      0.000000      0.000000
     2   0 1 1      0.000000      0.000000      0.000000
     2   0 0 2     -8.663739      0.000000     14.968184


 Parallel integral file used       8 records with       0 large values

                   NWChem Extensible Many-Electron Theory Module
                   ---------------------------------------------

              ======================================================
                   This portion of the program was automatically
                  generated by a Tensor Contraction Engine (TCE).
                  The development of this portion of the program
                 and TCE was supported by US Department of Energy,
                Office of Science, Office of Basic Energy Science.
                      TCE is a product of Battelle and PNNL.
              Please cite: S.Hirata, J.Phys.Chem.A 107, 9887 (2003).
              ======================================================

            General Information
            -------------------
      Number of processors :     8
         Wavefunction type : Restricted Hartree-Fock
          No. of electrons :    14
           Alpha electrons :     7
            Beta electrons :     7
           No. of orbitals :    60
            Alpha orbitals :    30
             Beta orbitals :    30
        Alpha frozen cores :     2
         Beta frozen cores :     2
     Alpha frozen virtuals :     0
      Beta frozen virtuals :     0
         Spin multiplicity : singlet 
    Number of AO functions :    30
       Number of AO shells :    12
        Use of symmetry is : on 
      Symmetry adaption is : on 
         Schwarz screening : 0.10D-09

          Correlation Information
          -----------------------
          Calculation type : Coupled-cluster singles & doubles                           
   Perturbative correction : none                                                        
            Max iterations :      100
        Residual threshold : 0.10D-06
     T(0) DIIS level shift : 0.00D+00
     L(0) DIIS level shift : 0.00D+00
     T(1) DIIS level shift : 0.00D+00
     L(1) DIIS level shift : 0.00D+00
     T(R) DIIS level shift : 0.00D+00
     T(I) DIIS level shift : 0.00D+00
   CC-T/L Amplitude update :  5-th order DIIS
                I/O scheme : Global Array Library

            Memory Information
            ------------------
          Available GA space size is    1677720700 doubles
          Available MA space size is     209713521 doubles

 Maximum block size        76 doubles

 tile_dim =      6

 Block   Spin    Irrep     Size     Offset   Alpha
 -------------------------------------------------
   1    alpha     ag     2 doubles       0       1
   2    alpha     b1u    1 doubles       2       2
   3    alpha     b2u    1 doubles       3       3
   4    alpha     b3u    1 doubles       4       4
   5    beta      ag     2 doubles       5       1
   6    beta      b1u    1 doubles       7       2
   7    beta      b2u    1 doubles       8       3
   8    beta      b3u    1 doubles       9       4
   9    alpha     ag     5 doubles      10       9
  10    alpha     au     1 doubles      15      10
  11    alpha     b1g    1 doubles      16      11
  12    alpha     b1u    6 doubles      17      12
  13    alpha     b2g    3 doubles      23      13
  14    alpha     b2u    2 doubles      26      14
  15    alpha     b3g    3 doubles      28      15
  16    alpha     b3u    2 doubles      31      16
  17    beta      ag     5 doubles      33       9
  18    beta      au     1 doubles      38      10
  19    beta      b1g    1 doubles      39      11
  20    beta      b1u    6 doubles      40      12
  21    beta      b2g    3 doubles      46      13
  22    beta      b2u    2 doubles      49      14
  23    beta      b3g    3 doubles      51      15
  24    beta      b3u    2 doubles      54      16

 Global array virtual files algorithm will be used

 Parallel file system coherency ......... OK

 #quartets = 3.081D+03 #integrals = 3.370D+04 #direct =  0.0% #cached =100.0%


 Integral file          = ./tce_n2.aoints.0
 Record size in doubles =  65536        No. of integs per rec  =  43688
 Max. records in memory =      2        Max. records in file   = 169376
 No. of bits per label  =      8        No. of bits per value  =     64


File balance: exchanges=     0  moved=     0  time=   0.0


 Fock matrix recomputed
 1-e file size   =              136
 1-e file name   = ./tce_n2.f1         
 Cpu & wall time / sec            0.0            0.1

 tce_ao2e: fast2e=1
 half-transformed integrals in memory

 2-e (intermediate) file size =         1497600
 2-e (intermediate) file name = ./tce_n2.v2i        
 Cpu & wall time / sec            0.1            0.1

 tce_mo2e: fast2e=1
 2-e integrals stored in memory

 2-e file size   =           111392
 2-e file name   = ./tce_n2.v2         
 Cpu & wall time / sec            0.3            0.3
 do_pt =   F
 do_lam_pt =   F
 do_cr_pt =   F
 do_lcr_pt =   F
 do_2t_pt =   F
 T1-number-of-tasks                        4

 t1 file size   =               20
 t1 file name   = ./tce_n2.t1         
 t1 file handle =       -999
 T2-number-of-boxes                      184

 t2 file size   =             2776
 t2 file name   = ./tce_n2.t2         
 t2 file handle =       -996

 CCSD iterations
 -----------------------------------------------------------------
 Iter          Residuum       Correlation     Cpu    Wall    V2*C2
 -----------------------------------------------------------------
    1   0.1547971890960  -0.3093800718205     0.4     0.4     0.0
    2   0.0428652446504  -0.3018484883219     0.4     0.4     0.0
    3   0.0157257630196  -0.3108072056048     0.4     0.4     0.0
    4   0.0077542456821  -0.3110010269719     0.4     0.4     0.0
    5   0.0038954443052  -0.3118169783090     0.4     0.4     0.0
 MICROCYCLE DIIS UPDATE:                        5                        5
    6   0.0006791528205  -0.3121843980447     0.4     0.4     0.0
    7   0.0002305946155  -0.3122330964657     0.4     0.4     0.0
    8   0.0001089729902  -0.3122330143678     0.4     0.4     0.0
    9   0.0000556441666  -0.3122342013884     0.4     0.4     0.0
   10   0.0000295530732  -0.3122352575442     0.4     0.4     0.0
 MICROCYCLE DIIS UPDATE:                       10                        5
   11   0.0000035883090  -0.3122371889841     0.4     0.4     0.0
   12   0.0000012484054  -0.3122369299847     0.4     0.4     0.0
   13   0.0000005316908  -0.3122370947895     0.4     0.4     0.0
   14   0.0000002683692  -0.3122371023894     0.4     0.4     0.0
   15   0.0000001348532  -0.3122371219224     0.4     0.4     0.0
 MICROCYCLE DIIS UPDATE:                       15                        5
   16   0.0000000223114  -0.3122371342529     0.4     0.4     0.0
 -----------------------------------------------------------------
 Iterations converged
 CCSD correlation energy / hartree =        -0.312237134252862
 CCSD total energy / hartree       =      -109.267389342369400

 Singles contributions

 Doubles contributions
     9b2g (alpha)     9b2g (beta ) ---     6b3u (alpha)     6b3u (beta )       -0.1032411464
     8b3g (alpha)     8b3g (beta ) ---     7b2u (alpha)     7b2u (beta )       -0.1032411464

 Parallel integral file used       8 records with       0 large values


 Task  times  cpu:        7.2s     wall:        7.8s


                                NWChem Input Module
                                -------------------


 Summary of allocated global arrays
-----------------------------------
  No active global arrays



                         GA Statistics for process    0
                         ------------------------------

       create   destroy   get      put      acc     scatter   gather  read&inc
calls:  833      833     8.53e+04 1715     7471        0        0        0     
number of processes/call 1.03e+00 1.25e+00 1.15e+00 0.00e+00 0.00e+00
bytes total:             1.29e+07 2.95e+06 1.64e+06 0.00e+00 0.00e+00 0.00e+00
bytes remote:            8.65e+06 1.83e+06 1.11e+06 0.00e+00 0.00e+00 0.00e+00
Max memory consumed for GA by this process: 1610928 bytes

MA_summarize_allocated_blocks: starting scan ...
MA_summarize_allocated_blocks: scan completed: 0 heap blocks, 0 stack blocks
MA usage statistics:

	allocation statistics:
					      heap	     stack
					      ----	     -----
	current number of blocks	         0	         0
	maximum number of blocks	        18	        29
	current total bytes		         0	         0
	maximum total bytes		   1060632	  22509576
	maximum total K-bytes		      1061	     22510
	maximum total M-bytes		         2	        23


                                     CITATION
                                     --------
                Please cite the following reference when publishing
                           results obtained with NWChem:

                 M. Valiev, E.J. Bylaska, N. Govind, K. Kowalski,
              T.P. Straatsma, H.J.J. van Dam, D. Wang, J. Nieplocha,
                        E. Apra, T.L. Windus, W.A. de Jong
                 "NWChem: a comprehensive and scalable open-source
                  solution for large scale molecular simulations"
                      Comput. Phys. Commun. 181, 1477 (2010)
                           doi:10.1016/j.cpc.2010.04.018

                              AUTHORS & CONTRIBUTORS
                              ----------------------
      E. J. Bylaska, W. A. de Jong, N. Govind, K. Kowalski, T. P. Straatsma,
     M. Valiev, H. J. J. van Dam, D. Wang, E. Apra, T. L. Windus, J. Hammond,
     J. Autschbach, P. Nichols, S. Hirata, M. T. Hackler, Y. Zhao, P.-D. Fan,
      R. J. Harrison, M. Dupuis, D. M. A. Smith, K. Glaesemann, J. Nieplocha,
      V. Tipparaju, M. Krishnan, A. Vazquez-Mayagoitia, L. Jensen, M. Swart,
      Q. Wu, T. Van Voorhis, A. A. Auer, M. Nooijen, L. D. Crosby, E. Brown,
             G. Cisneros, G. I. Fann, H. Fruchtl, J. Garza, K. Hirao,
        R. Kendall, J. A. Nichols, K. Tsemekhman, K. Wolinski, J. Anchell,
       D. Bernholdt, P. Borowski, T. Clark, D. Clerc, H. Dachsel, M. Deegan,
        K. Dyall, D. Elwood, E. Glendening, M. Gutowski, A. Hess, J. Jaffe,
        B. Johnson, J. Ju, R. Kobayashi, R. Kutteh, Z. Lin, R. Littlefield,
    X. Long, B. Meng, T. Nakajima, S. Niu, L. Pollack, M. Rosing, G. Sandrone,
       M. Stave, H. Taylor, G. Thomas, J. H. van Lenthe, A. Wong, Z. Zhang.

 Total times  cpu:        7.2s     wall:        9.6s
